src/clj/rems/css/styles.clj

```diff
@@ -571,24 +613,6 @@
                             :bottom 0
                             :right (u/px 140)}]])
 
-   ;; Logo, login, etc.
-   [:.logo {:height (u/px 140)
-            :background-color (get-theme-attribute :logo-bgcolor)
-            :padding "0 20px"
-            :margin-bottom (u/em 1)}]
-   [(s/descendant :.logo :.img) {:height "100%"
-                                 :background-color (get-theme-attribute :logo-bgcolor)
-                                 :background-image (get-logo-image context/*lang*)
-                                 :-webkit-background-size :contain
-                                 :-moz-o-background-size :contain
-                                 :-o-background-size :contain
-                                 :background-size :contain
-                                 :background-repeat :no-repeat
-                                 :background-position [[:center :center]]
-                                 :background-origin (get-theme-attribute :logo-content-origin)
-                                 :padding-left (u/px 20)
-                                 :padding-right (u/px 20)
-                                 :padding-top (u/px 8)}]
```

Comment:
Where has this padding gone? Is it not needed?

Reply:
probably, if I removed it? Would you like me to go and double check? Sure, I will.

Reply:
I did check, yes, all good